lib/senkyoshi/models/root_resource.rb

```diff
@@ -0,0 +1,23 @@
+require "senkyoshi/models/resource"
+
+module Senkyoshi
+  ##
+  # Class to represent a resource constructed from a single 'dat' file.
+  ##
+  class RootResource < Resource
```

Comment:
Yeah I was having a real hard time coming up for a good name. Do you have any other ideas for a name? Something that captures that it is created from a single .dat file? I am not super happy with RootResource.